content/en/references/credentials.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Credentials"
+linkTitle: "Credentials"
+categories: ["LocalStack"]
+tags: ["access-key-id", "secret-access-key", "account-id"]
+weight: 5
+description: >
+  Credentials for accessing LocalStack services
+---
+
+Like AWS, LocalStack requires access key IDs to be set in all operations.
+The choice of access key ID will affect [multi-account namespacing]({{< ref "multi-account-setups" >}}).
+Values of secret access keys are currently ignored by LocalStack.
+
+Access key IDs can be one of following patterns:
+
+### Accounts IDs
+
+You can specify a 12-digit number which will be taken by LocalStack as the account ID.
+For example, `112233445566`.
+
+### Structured access key ID
+
+You can specify a structured key like `LSIAQAAAAAAVNCBMPNSG` (which translates to account ID `000000000042`).
+This must be at least 20 characters in length and must be decodable to an account ID.
+
+{{< alert title="Note">}}
+In the future LocalStack will support IAM service which will issue proper access key IDs.
+{{< /alert >}}
```

Comment:
We already issue proper access key ids?

Reply:
@dfangl: The note was referring to initial process of getting credentials but I admit it's confusing. Removed with 06630404 :+1: